e2e/cypress/integration/messaging/permalink_message_edit_spec.js

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// [number] indicates a test step (e.g. # Go to a page)
+// [*] indicates an assertion (e.g. * Check the title)
+// Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Permalink message edit', () => {
+    it('M18717 - Edit a message in permalink view', () => {
+        // # Login as "user-1" and go to /
+        cy.apiLogin('user-1');
+        cy.visit('/');
+
+        const searchWord = 'searchtest';
```

Comment:
Not mentioned in the ticket but please make this random.
```suggestion
        const searchWord = `searchtest ${Date.now()}`;
```

Reply:
Good idea, to make it more explicit 👍